DemoApp/Sources/Swift6Migration/CustomStringConvertible+Conformances.swift

```diff
@@ -0,0 +1,152 @@
+//
+// Copyright © 2025 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import StreamVideo
+
+#if compiler(>=6.0)
+extension TranscriptionSettings.Mode: @retroactive CustomStringConvertible {
+    public var description: String {
+        switch self {
+        case .autoOn:
+            return "auto-on"
+        case .available:
+            return "Available"
+        case .disabled:
+            return "Disabled"
+        case .unknown:
+            return "Unknown"
+        }
+    }
+}
+
+extension TranscriptionSettings.ClosedCaptionMode: @retroactive CustomStringConvertible {
+    public var description: String {
+        switch self {
+        case .autoOn:
+            return "auto-on"
+        case .available:
+            return "Available"
+        case .disabled:
+            return "Disabled"
+        case .unknown:
+            return "Unknown"
+        }
+    }
+}
+
+extension TranscriptionSettings.Language: @retroactive CustomStringConvertible {
+    public var description: String {
```

Comment:
This is a bit repetitive and will be tricky to maintain. Why not have just empty extension that provides the description, while you do the conformance like it is now:
```
#if compiler(>=6.0)
extension TranscriptionSettings.Language: @retroactive CustomStringConvertible {}
#else 
extension TranscriptionSettings.Language: CustomStringConvertible {}
#endif 
```
And then:
```
extension TranscriptionSettings.Language {
    public var description: String {
      ...
    }
```

Reply:
You are absolutely right! Thanks